ursapictura/ThriveWell-Server
# Repository

## Detailed Description
Fleshed out core capabilities of Repository layer and Endpoints to reflect the CRUD requirements of the project. All API calls have been tested in Swagger without issue.


## How to Test
1.  pull down "repository" branch to local machine
2.  ensure .NET has been installed
3.  add User Secrets for DbContext
4. run Swagger to test API calls

## Types of Changes
